data/json/items/gun/30carbine.json

```diff
@@ -0,0 +1,78 @@
+[
+  {
+    "id": "m1carbine",
+    "copy-from": "rifle_semi",
+    "looks_like": "garand",
+    "type": "GUN",
+    "name": { "str": "M1 carbine" },
+    "description": "A light rifle developed during the Second World War to provide a less cumbersome weapon for soldiers that didn't need the firepower of an M1 Garand. Firing the .30 Carbine cartridge, it provides more firepower than a pistol and is easier to carry than most other rifles.",
```

Comment:
```suggestion
    "description": "A light rifle developed during the Second World War to provide a less cumbersome weapon for soldiers that didn't need the firepower of a M1 Garand. Firing the .30 Carbine cartridge, it provides more firepower than a pistol and is easier to carry than most other rifles.",
```

Reply:
I find the style ambiguous on this one. In this context the "M" in "M1" is pronounced with an "em" vowelly sound, so I was inclined toward "an". 
To avoid this entirely, changing to "the".